src/Storages/MergeTree/MergeTreeData.cpp

```diff
@@ -1252,7 +1252,16 @@ void MergeTreeData::dropAllData()
     clearPartsFromFilesystem(all_parts);
 
     for (const auto & [path, disk] : getRelativeDataPathsWithDisks())
-        disk->removeRecursive(path);
+    {
+        try
+        {
+            disk->removeRecursive(path);
+        }
+        catch (const Poco::FileNotFoundException &)
+        {
+            /// If the file is already deleted, do nothing.
```

Comment:
```suggestion
            /// If the file is already deleted, do nothing.
            tryLogCurrentException(log);
```

Something probably went wrong if file should be here, but was not found, so why not to log the error.